sw/vendor/patches/mundane/build_with_bazel.patch

```diff
@@ -0,0 +1,153 @@
+diff --git a/.gitignore b/.gitignore
+index aa085cd..e16558b 100644
+--- a/.gitignore
++++ b/.gitignore
+@@ -1,3 +1,4 @@
+ Cargo.lock
+ /target
+ **/*.rs.bk
++bazel-*
+diff --git a/BUILD b/BUILD
+new file mode 100644
+index 0000000..651c42d
+--- /dev/null
++++ b/BUILD
+@@ -0,0 +1,54 @@
++load("@rules_rust//rust:defs.bzl", "rust_library", "rust_test")
++load("@boringssl//:prefix.bzl", "boringssl_prefixed_library")
++
++package(default_visibility = ["//visibility:public"])
++
++boringssl_prefixed_library(
++    name = "crypto",
++    library = "@boringssl//:prefixed_crypto",
++    symbol_prefix = "__RUST_MUNDANE_0_4_4",
```

Comment:
I think we can avoid all of this by just zapping all of the `#[link_name]`s in `boringssl.rs`. The symbol-renaming stuff is, AIUI, distinctly in "keep both pieces" territory.

Reply:
I'm ambivalent about the symbol renaming.  However, `boringssl/boringssl.rs` is auto-generated by bindgen and then massaged by some shell scripts.  Eliminating the `#[link name]`s means I'd have to go dig into that rather than using the version that is already committed to the repo.

Reply:
I've reworked this to avoid symbol renaming.

The current `build_with_bazel.patch` no longer requires a patched borginssl.